packages/airnode-deployer/src/infrastructure/index.test.ts

```diff
@@ -1521,3 +1534,288 @@ describe('deploymentInfo', () => {
     expect(gcpGetFileFromBucketSpy).not.toHaveBeenCalled();
   });
 });
+
+describe('fetchFiles', () => {
+  const bucket = {
+    name: 'airnode-123456789',
+    region: 'us-east-1',
+  };
+  const configPath = path.join(__dirname, '..', '..', 'test', 'fixtures', 'config.aws.valid.json');
+  const secretsPath = path.join(__dirname, '..', '..', 'test', 'fixtures', 'secrets.valid.env');
+  const directoryStructure = pick(mockBucketDirectoryStructure, [
+    '0xd0624E6C2C8A1DaEdE9Fa7E9C409167ed5F256c6',
+    '0xA30CA71Ba54E83127214D3271aEA8F5D6bD4Dace',
+  ]);
+
+  let tempOutputDirectory: string;
+  let awsGetAirnodeBucketSpy: jest.SpyInstance;
+  let awsGetBucketDirectoryStructureSpy: jest.SpyInstance;
+  let awsGetFileFromBucketSpy: jest.SpyInstance;
+
+  beforeEach(() => {
+    tempOutputDirectory = fs.mkdtempSync(path.join(os.tmpdir(), 'airnode-deployer-fetchFiles-test'));
+    awsGetAirnodeBucketSpy = jest.spyOn(aws, 'getAirnodeBucket').mockResolvedValue(bucket);
+    awsGetBucketDirectoryStructureSpy = jest
+      .spyOn(aws, 'getBucketDirectoryStructure')
+      .mockResolvedValue(directoryStructure);
+    awsGetFileFromBucketSpy = jest.spyOn(aws, 'getFileFromBucket').mockImplementation((_bucket, path) => {
+      if (path.includes('config.json')) {
+        return Promise.resolve(fs.readFileSync(configPath).toString());
+      }
+      if (path.includes('secrets.env')) {
+        return Promise.resolve(fs.readFileSync(secretsPath).toString());
+      }
+
+      throw new Error(`Mocking fetching of unsupported file '${path}'`);
+    });
+  });
+
+  afterEach(() => {
+    fs.rmSync(tempOutputDirectory, { recursive: true });
+  });
+
+  it('fetch files for deployment', async () => {
+    const deploymentId = 'aws7195b548';
+    const fetchedFilename = `${deploymentId}-3580a278.zip`;
```

Comment:
I wonder why the filename is not just `${deploymentId}.zip`.

Reply:
Because even though you don't fetch the specific version in this test a specific version is fetched (the latest one). And in case you would do `airnode-deployer fetch-files aws7195b548` and `airnode-deployer fetch-files aws7195b548 3580a278` I didn't want it to store the result in two different files because it's the same thing.

Reply:
Maybe it could be called `${deploymentId}-latest.zip`, but what you did makes sense as well.